item.rb

```diff
@@ -0,0 +1,18 @@
+class Item
+
+  attr_reader :name, :price, :weight
+  attr_writer :name, :price, :weight
+  
+  def initialize (name, price, weight)
+    @name = name
+    @price = price
+    @weight = weight
+  end
+end
+
+
+item_1 = Item.new("white_bread", "0.5$", "400g")
```

Comment:
аргументы price и weight подразумеваются как флоаты. В следующих уроках мы будем писать методы с вычислениями, а строки не вычисляются
```suggestion
item_1 = Item.new("white_bread", 0.5, 400.0)
```